go/cmd/vtctldclient/internal/command/vschemas.go

```diff
@@ -33,8 +39,117 @@ var (
 		Args: cobra.ExactArgs(1),
 		RunE: commandGetVSchema,
 	}
+	// ApplyVSchema makes an ApplyVSchema gRPC call to a vtctld.
+	ApplyVSchema = &cobra.Command{
```

Comment:
A net-new thing (that you could not have known!) I'm trying to introduce is:
* Set `DisableFlagsInUseLine: true` on every command. This will change the help from `{{ .Use }} [flags]` to just `{{ .Use }}` (which is better, because you include the actual named flags in the Use string).
* Include a `.Short` help text, which in most cases I just copy from the old vtctl command.